src/sonic-yang-models/yang-models/sonic-wred-profile.yang

```diff
@@ -0,0 +1,190 @@
+module sonic-wred-profile {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-wred-profile";
+
+    prefix wrd;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "WRED_PROFILE yang Module for SONiC OS";
+
+    revision 2021-04-01 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-wred-profile {
+
+        container WRED_PROFILE {
+
+            description "WRED_PROFILE part of config_db.json";
+
+            list WRED_PROFILE_LIST {
+
+                key "name";
+
+         //       sonic-ext:custom-validation ValidateAtLeastOneColorEnabled;
+
+                leaf name {
+                    type string {
+                        pattern '[a-zA-Z0-9]{1}([-a-zA-Z0-9_]{0,31})';
+                        length 1..32;
+                    }
+                    description "WRED profile name";
+                }
+
+                leaf yellow_min_threshold {
+                    type uint64;
+                    units bytes;
+                    default 0;
+                    description "Yellow minimum threshold bytes";
+                }
+
+                leaf green_min_threshold {
+                    type uint64;
+                    units bytes;
+                    default 0;
```

Comment:
@AshokDaparthi : As discussed in the yang subgroup community call, please remove the default values from all these yang attributes.

Reply:
Done